# 2. EDA - overview

In [22]:
df_modelo.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 6401 entries, 0 to 6400
Data columns (total 28 columns):
 #   Column                       Non-Null Count  Dtype   
---  ------                       --------------  -----   
 0   cod_mun                      6401 non-null   object  
 1   año                          6401 non-null   object  
 2   NOMBRE                       6400 non-null   object  
 3   analfabetismo                6400 non-null   float64 
 4   bajo_logro_educativo         6400 non-null   float64 
 5   barreras_primera_infancia    6400 non-null   float64 
 6   barreras_acceso_salud        6400 non-null   float64 
 7   tasa_dependencia             6400 non-null   float64 
 8   hacinamiento_critico         6400 non-null   float64 
 9   inadec_eliminacion_excretas  6400 non-null   float64 
 10  inasistencia_escolar         6400 non-null   float64 
 11  inadec_paredes               6400 non-null   float64 
 12  inadec_pisos                 6400 non-null   float64 
 13  rez

## 2.1 Duplicados

In [23]:
df_modelo.duplicated(keep = 'first').sum()

# No hay valores duplicados

np.int64(0)

## 2.2 Datos númericos

In [24]:
df_modelo.describe().round(1).T

,count,mean,std,min,25%,50%,75%,max
analfabetismo,6400.0,17.4,8.3,2.0,11.5,16.1,22.2,49.5
bajo_logro_educativo,6400.0,65.7,13.1,18.3,59.0,69.0,75.3,96.2
barreras_primera_infancia,6400.0,2.7,2.9,0.4,1.4,1.9,2.7,34.6
barreras_acceso_salud,6400.0,5.2,3.8,0.0,2.7,4.3,6.6,40.2
tasa_dependencia,6400.0,40.8,12.5,14.7,31.9,39.4,48.1,97.7
hacinamiento_critico,6400.0,11.6,7.5,1.4,5.9,9.2,16.2,58.1
inadec_eliminacion_excretas,6400.0,25.1,22.6,0.2,7.5,17.6,35.7,98.7
inasistencia_escolar,6400.0,5.3,3.4,0.9,3.1,4.5,6.3,42.8
inadec_paredes,6400.0,5.7,8.5,0.0,0.7,2.6,7.3,72.7
inadec_pisos,6400.0,15.7,16.8,0.0,2.8,9.6,23.0,86.7


*   Trabajo Informal: El promedio nacional municipal es alarmante: 87.4%, llegando al 100% en los municipios más vulnerables. Esto demuestra que la informalidad laboral es un factor estructural casi universal en los municipios del país.
*   Bajo Logro Educativo: Tiene un promedio de 65.7% y alcanza picos de 96.2%.
*   Acceso a Agua Potable (sin_agua_mejorada): Hay municipios donde prácticamente todos tienen acceso (0.2% de carencia), mientras que en otros el 99% de la población carece de agua limpia. Esta brecha justifica por qué la desnutrición se dispara en ciertas regiones.
*   El 50% de los municipios tiene 1,434 o menos niños entre 0 y 4 años (mediana), pero el valor máximo llega a 478,485 niños (Bogotá o grandes capitales).